scripts/show_history.py

```
@@ -0,0 +1,173 @@
+        if not os.path.exists(self.history_file):
+            return {}
+            
+        with open(self.history_file, 'r') as f:
+            return json.load(f)
+    
+    def format_timestamp(self, timestamp: str) -> str:
+        """Format ISO timestamp to human readable"""
+        try:
+            dt = datetime.fromisoformat(timestamp)
+            return dt.strftime("%Y-%m-%d %H:%M:%S")
+        except:
```

Flake8 (7.2.0)

[error] do not use bare 'except'

(E722)